### DISCUSSION TITLE
[HELP] About Example: pendulum_3d.py

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#840] [Q&A] [answered]
Hi Marco,
I downloaded the pendulum_3d.py from website and run it. But the window was not showed. Here is the errors.
Traceback (most recent call last):
  File "g:\BaiduNetdiskWorkspace\XVedo\test\pendulum_anim.py", line 25, in <module>
    for b1, b2 in progressbar(zip(p1,p2)):
                  ^^^^^^^^^^^
NameError: name 'progressbar' is not defined. Did you mean: 'ProgressBar'?

My env:
os: windows10
vedo: 2023.4.3

Thanks!

**Comment by marcomusy** [comment, marked as answer]
hi, all the examples in the github links always refer to the latest dev version:
```
pip install -U git+https://github.com/marcomusy/vedo.git
```

OR, without updating, in a terminal type
```
vedo --run pendulum_3d
```

**Reply by ZiguoAtGitHub** [reply]
Got it!
Thanks!
